### DISCUSSION TITLE
خطای درگاه آسان پرداخت در webforms

### DISCUSSION THREAD

**Discussion by saeed-m** [Sina-Soltani/Parbad#300] [Q&A]
در مثالی که برای وب فرم تهیه کردید  وقتی درگاه آسان پرداخت انتخاب میشه ب خطای 
Parbad.Exceptions.GatewayNotFoundException: 'No gateway found with the name AsanPardakht. Make sure you have already added the required gateways. To add and configure a gateway use the ConfigureGateways method.'

**Comment by Sina-Soltani** [comment]
متن این خطا واضح نیست؟

**Comment by saeed-m** [comment]
بله لطف میکنید نحوه پیاده سازیشم بفرمایید.

**Reply by Sina-Soltani** [reply]
لطفا با دقت بیشتری به صفحه ها و مطالب توجه کنید.

https://github.com/Sina-Soltani/Parbad.Samples

**Comment by saeed-m** [comment]
ممنون پیاده سازی انجام شد.
مشکلی که الان وجود داره هنگام بازگشت از دروازه پرداخت هست.
که متذ 
var invoice = await StaticOnlinePayment.Instance.FetchAsync();
با خطای 
Request timed out.
مواجه میشه.
که در دریافت و خواندن توکن به مشکل بر می خوره

**Comment by Sina-Soltani** [comment]
چه نوگت هایی و چه نسخه هایی رو نصب کردید؟

**Comment by saeed-m** [comment]
Parbad 3.8.3
Parbad.Storage.Abstractions 1.0.0
Parbad.Owin 1.2.0
Parbad.Storage.Cache 1.2.0
